addons/mail/static/src/models/call_sidebar_view.js

```diff
@@ -1,16 +1,38 @@
 /** @odoo-module **/
 
 import { registerModel } from '@mail/model/model_core';
-import { one } from '@mail/model/model_field';
+import { one, many } from '@mail/model/model_field';
+import { insertAndReplace, replace } from '@mail/model/model_field_command';
 
 registerModel({
     name: 'CallSidebarView',
     identifyingFields: ['callView'],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeSidebarTiles() {
+            const tiles = [];
+            for (const channelMember of this.callView.channel.channel.callParticipants) {
+                if (this.callView.filterVideoGrid && !channelMember.isStreaming) {
+                    continue;
+                }
+                tiles.push({ channelMember: replace(channelMember) });
+            }
+            return insertAndReplace(tiles);
```

Comment:
well this compute is the same as the other one, so we should probably definitely have another field now on a common model to compute only once the list of participants to display